src/config.rs

```diff
@@ -298,12 +310,13 @@ fn test_amend_preserve_password_hashes() {
 	let initial_config = Config {
 		album_art_pattern: None,
 		reindex_every_n_seconds: None,
+		prefix_url: None,
```

Comment:
Can you add a value for prefix_url in the new_config so that editing it gets tested?